party-venues.md

```diff
@@ -3,3 +3,7 @@
 - Congress Park
 - My apartment
 - The Blakement
+
+- Mexico
+- Any Beach Anywhere
```

Comment:
Well... I don't really think it needs to be specific, but I will change it for you.